src/ui/components/Account/Library.tsx

```diff
@@ -57,12 +57,7 @@ function Library({ setWorkspaceId, setModal, currentWorkspaceId }: PropsFromRedu
   }, [workspaces, loading]);
 
   useEffect(() => {
-    if (
-      userInfo &&
-      userInfo.authoredRecordingCount === 0 &&
-      userInfo.nags &&
-      !userInfo.nags.includes(Nag.FIRST_REPLAY)
-    ) {
+    if (userInfo?.nags && !userInfo.nags.includes(Nag.FIRST_REPLAY)) {
```

Comment:
I ran some SQL on the database to set FIRST_REPLAY for all users who have recordings, so the only users without `FIRST_REPLAY` in their nags at this point should be users who have no recordings.

Reply:
For history's sake, this is what I ran:
```
UPDATE users SET nags = nags||'["first_replay"]'::jsonb WHERE EXISTS (SELECT * FROM recordings WHERE users.id = recordings.user_id) AND NOT (nags ?| array['first_replay']);
```